docs/content/howto/ros2-nav-turtlebot.md

```diff
@@ -415,27 +411,30 @@ if node_data:
     # Log the transform between this node and its direct parent (if it has one!).
     if parent:
         world_from_mesh = node_data[0]
-        t = trimesh.transformations.translation_from_matrix(world_from_mesh)
-        q = trimesh.transformations.quaternion_from_matrix(world_from_mesh)
-        # `trimesh` stores quaternions in `wxyz` format, rerun needs `xyzw`
-        # TODO(jleibs): Remove conversion once
-        # [#883](https://github.com/rerun-io/rerun/issues/883) is closed
-        q = np.array([q[1], q[2], q[3], q[0]])
-        rr.log_transform3d(path,
-            rr.TranslationRotationScale3D([t.x, t.y, t.z], rr.Quaternion(xyzw=q), timeless=timeless)
+        rr.log(
+            path,
+            rr.Translation3D(
+                rr.TranslationAndMat3(
+                    translation=trimesh.transformations.translation_from_matrix(world_from_mesh),
+                    mat3x3=world_from_mesh[0:3, 0:3],
+                )
+            ),
```

Comment:
Why not 
```
rr.Translation3D(translation=.., mat3x3=...)
```

Reply:
Ahh that didn't exist when someone ported the example, which I copied over from.